src/miner.app.src

```diff
@@ -26,9 +26,11 @@
     xmerl,  %% nat_upnp has a missing dep
     jsx,
     kvc,
-    longfi
+    longfi,
+    grpcbox,
+    grpc_lib
    ]},
-  {included_applications, [blockchain]},
+  {included_applications, [blockchain, sibyl]},
```

Comment:
are we starting this as an included app?  is there a way it can just be part of the regular supervision tree?

Reply:
Sibyl is defined as an included app and its sup is started as part of miner_restart_sup but only if the miner env var 'mode' is set to validator